client/containers/DashboardOverview/OverviewBlocks.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'components';
+
+require('./overviewBlocks.scss');
+
+const propTypes = {
+	overviewData: PropTypes.object.isRequired,
+	useCollectionPubsObject: PropTypes.object.isRequired,
+};
+
+const OverviewBlocks = (props) => {
+	const { overviewData, useCollectionPubsObject } = props;
+	const { collectionPubs } = useCollectionPubsObject;
+
+	const countItems = (type) => {
+		const activeList = collectionPubs ? collectionPubs.map((cp) => cp.pub) : overviewData.pubs;
+		return activeList.reduce((prev, curr) => {
```

Comment:
One thing I find improves readability of `reduce` calls is naming the arguments to the reducer something other than `(prev, curr)` so we can tell what they represent — in this case I'd suggest `(count, pub)`.

Reply:
Happy to try that. I find with reducers, keeping the logic of how the loop is iterating can be really helpful. Sometimes seeing `prev` is the most useful name for me because I know it's the thing that got passed on from the last loop. In more complex reducers with many variables, `count` may not stand out as loudly. That could just be how I've become used to writing reducers more than sound logic - and for this simple case I agree it reads cleaner with `count, pub`.